### PR TITLE
feat: create and enable user to complete agora first vote action

### PR DESCRIPTION
**Pull request by juliano-quatrin-nunes**
Closes [#355](https://linear.app/bleu-builders/issue/OP-355/allow-user-to-complete-agora-first-vote-action-execution-type), [#353](https://linear.app/bleu-builders/issue/OP-353/create-agora-first-vote-action-execution-strategy)

Preview: 
image

### CODE REVIEW

File: apps/govquests-api/govquests/action_tracking/lib/action_tracking/strategies/verify_first_vote.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+require_relative "base"
+
+module ActionTracking
+  module Strategies
+    class VerifyFirstVote < Base
+      def on_start_execution
+        {
+          proposalsVotedOn: agora_delegate["proposalsVotedOn"].to_i
```
Comment (ribeirojose) [inline apps/govquests-api/govquests/action_tracking/lib/action_tracking/strategies/verify_first_vote.rb:8]: any reason to use camelCase here? 🤔 I think we have as a standard using snake_case in these hashes' keys

File: apps/govquests-frontend/next.config.mjs [hunk 2]
```diff
@@ -8,7 +8,7 @@ const nextConfig = {
   },
   async rewrites() {
     const destination =
-      process.env.NEXT_PUBLIC_API_URL || "http://localhost:3001";
+      process.env.NEXT_PUBLIC_API_URL || "http://localhost:3001/graphql";
```
Comment (ribeirojose) [inline apps/govquests-frontend/next.config.mjs:11]: 🙏

**Review by ribeirojose — APPROVED** [review]
ty for the fixes 😬

File: apps/govquests-api/govquests/action_tracking/lib/action_tracking/strategies/verify_first_vote.rb (hunk 1 above)
Comment (juliano-quatrin-nunes) [reply apps/govquests-api/govquests/action_tracking/lib/action_tracking/strategies/verify_first_vote.rb:8]: No reason at all, I will change!!

**Pull request merged**